.github/workflows/docker-publish.yml

```diff
@@ -69,13 +69,13 @@ jobs:
 
           make -C kind install
 
-      - name: Build kind-node image - ${{ matrix.version }}
+      - name: Build kind-node image - ${{ inputs.k8s-version }}
         run: |
           mkdir -p tmp/kubernetes
-          git clone --single-branch --filter=tree:0 --branch ${{ matrix.version }} https://github.com/kubernetes/kubernetes tmp/kubernetes
-          kind build node-image $PWD/tmp/kubernetes --arch ppc64le --image ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ matrix.version }} --base-image ${{ env.BASE_IMAGE }}
+          git clone --single-branch --filter=tree:0 --branch ${{ inputs.k8s-version }} https://github.com/kubernetes/kubernetes tmp/kubernetes
+          kind build node-image $PWD/tmp/kubernetes --arch ppc64le --image ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ inputs.k8s-version }} --base-image ${{ env.BASE_IMAGE }}
 
       - name: Publish node image
-        if: github.event_name != 'pull_request'
+        if: github.event_name != 'pull_request' || github.event_name != 'push'
```

Comment:
can we add comment why are we not pushing the image for any commit?

Reply:
Done!